frontend/src/pages/nft/nft.tsx

```diff
@@ -104,47 +126,79 @@ function NFT() {
-                </li>
-
-                <li className={styles.row}>
-                  <span>Creator Earnings:</span>
-                  <span className={styles.value}>
-                    {royalty !== undefined ? `${royalty}%` : <Skeleton width="25%" borderRadius="4px" />}
-                  </span>
-                </li>
-              </ul>
-            </div>
+            <Tabs
+              list={TABS}
+              size="small"
+              outlined
+              className={styles.tabs}
+              value={selectedTab}
+              onChange={onSelectedTabChange}
+            />
+
+            {selectedTab === 0 && (
+              <div className={styles.card}>
+                <h3 className={styles.heading}>
+                  <InfoSVG />
+                  Details
+                </h3>
+
+                <ul className={styles.table}>
+                  <li className={styles.row}>
+                    <span>Contract Address:</span>
+                    <span className={styles.value}>
+                      <span className={styles.address}>{collectionId}</span>
+                      <CopyButton value={collectionId} />
+                    </span>
+                  </li>
+
+                  <li className={styles.row}>
+                    <span>Token ID:</span>
+                    <span className={styles.value}>{id}</span>
+                  </li>
+
+                  <li className={styles.row}>
+                    <span>Token Standart:</span>
+                    <span className={styles.value}>gNFT</span>
+                  </li>
+
+                  <li className={styles.row}>
+                    <span>Mint Date:</span>
+                    <span className={styles.value}>
+                      {createdAt ? new Date(createdAt).toLocaleString() : <Skeleton width="25%" borderRadius="4px" />}
+                    </span>
+                  </li>
+
+                  <li className={styles.row}>
+                    <span>Creator Earnings:</span>
+                    <span className={styles.value}>
+                      {royalty !== undefined ? `${royalty}%` : <Skeleton width="25%" borderRadius="4px" />}
+                    </span>
+                  </li>
+                </ul>
+              </div>
+            )}
+
+            {selectedTab === 1 && (
+              <div className={styles.card}>
+                <h3 className={styles.heading}>
+                  <FeaturedPlayListSVG />
+                  NFT Details
+                </h3>
+
+                {!!parsedNFTMetadata?.length && (
+                  <ul className={styles.metadata}>
+                    {parsedNFTMetadata.map((value, index) => (
+                      <li className={styles.metadataItem} key={index}>
+                        <span className={styles.metadataContainer}>
+                          <MarkerSVG />
+                          <span className={styles.metadataText}>{value}</span>
+                        </span>
+                      </li>
```

Comment:
```suggestion
                      <li className={styles.metadataItem} key={index}>
                          <MarkerSVG />
                          <span className={styles.metadataText}>{value}</span>
                      </li>
```

Do you think we need `span` container there at all?